Tests/TutorLizard.BusinessLogic.Tests/Services/Student/StudentServiceScheduleItemRequestTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Moq;
+using TutorLizard.BusinessLogic.Interfaces.Data.Repositories;
+using TutorLizard.BusinessLogic.Models;
+using TutorLizard.BusinessLogic.Models.DTOs.Requests;
+using TutorLizard.BusinessLogic.Services;
+
+namespace TutorLizard.BusinessLogic.Tests.Services.Student
+{   
+    public class StudentServiceScheduleItemRequestTests : StudentServiceTestBase
+    {
+        [Fact]
+        public async Task CreateScheduleItemRequest_WhenIsRemoteIsTrue_ShouldSetIsRemoteToTrue()
+        {
+            // Arrange
+            var scheduleItem = new ScheduleItem { Id = 1, Ad = new Ad { TutorId = 2 } };
+            SetupMockGetScheduleItemById(scheduleItem);
+
+            var request = new CreateScheduleItemRequestRequest
+            {
+                StudentId = 2,
+                ScheduleItemId = 1,
+                IsRemote = true
+            };
+
+            // Act
+            await StudentService.CreateScheduleItemRequest(request);
+
+            // Assert
+            MockScheduleItemRequestRepository.Verify(repo => repo.Create(It.Is<ScheduleItemRequest>(req => req.IsRemote == true)), Times.Once);
```

Comment:
You don't specify what the `.Create()` returns. In our current implementation the test works, because the return value is not used, but if it was, you'd get an error. I recommend to put something like this in the `Arrange` step:
```
MockScheduleItemRequestRepository
    .Setup(x => x.Create(It.Is<ScheduleItemRequest>(req => req.IsRemote == true)))
    .Returns((ScheduleItemRequest req) => Task.FromResult(req))
    .Verifiable(Times.Once);
```
And then in `Assert` you can just do `MockScheduleItemRequestRepository.VerifyAll();`.